diff --git a/data/org.freedesktop.impl.portal.desktop.lxqt.service.in b/data/org.freedesktop.impl.portal.desktop.lxqt.service.in
index 5752eaa..2e83265 100644
--- a/data/org.freedesktop.impl.portal.desktop.lxqt.service.in
+++ b/data/org.freedesktop.impl.portal.desktop.lxqt.service.in
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=org.freedesktop.impl.portal.desktop.lxqt
-Exec=/bin/sh -c 'env QT_QPA_PLATFORMTHEME=lxqt @CMAKE_INSTALL_FULL_LIBEXECDIR@/xdg-desktop-portal-lxqt'
+Exec=@CMAKE_INSTALL_FULL_LIBEXECDIR@/xdg-desktop-portal-lxqt
